src/app/App.js

```diff
@@ -17,19 +17,40 @@ export const App = ({options}) => {
    //add event listener to the select mode menu button 
    const help=document.querySelector('#helpScreen')
    const style = getComputedStyle(help);
+   
+   const changeColorOfClickedButton = (querySelector) => {
+
+      const whoIsThatPokemonOption = document.querySelector('#whatItLooksLikeOption')
+      const whatItLooksLikeOption = document.querySelector('#guessTheTypeOption')
+      const guessTheTypeOption = document.querySelector('#whoIsThatPokemonOption')
+
+      whoIsThatPokemonOption.style.backgroundColor="#3762AC"
+      whoIsThatPokemonOption.style.color="#FFCB05"
+      whatItLooksLikeOption.style.backgroundColor="#3762AC"
+      whatItLooksLikeOption.style.color="#FFCB05"
+      guessTheTypeOption.style.backgroundColor="#3762AC"
+      guessTheTypeOption.style.color="#FFCB05"
+
+      querySelector.style.backgroundColor='#FFCB05'
```

Comment:
`querySelector` parameter should be only string i.e. `#whatItLooksLikeOption`. Then, inside of this function, you should select fourth button, i.e.
`const selectedOption = document.querySelec...`

and then change colors on it.